nailgun/entities.py

```diff
@@ -1042,6 +1042,7 @@ def __init__(self, server_config=None, **kwargs):
             'attach_stdin': entity_fields.BooleanField(),
             'attach_stdout': entity_fields.BooleanField(),
             'command': entity_fields.StringField(
+                default='top',
```

Comment:
Previously nailgun was sending just random string which would fail in 100% cases when command is not specified, so imo it's still a better option (and if user haven't specified some field it basically means he doesn't care what value will be provided and lets nailgun decide how to fill it, that's why we have `create_missing`). But i'm ok with defining any other command if you remember more widespread ones (pwd? cd? echo?)